lib/screens/finder.rb

```diff
@@ -55,12 +55,12 @@ def ask_to_expand
   end
 
   def expand_contact(matched)
-    index = ask_for_contact - 1
+    index = ask_for_contact_index - 1
```

Comment:
Should the `Finder` class know about the `- 1`?
Where does the need come from to subtract 1 from what ever a user enters?